### DISCUSSION TITLE
Large tables will go outside screen on iPhone.

### DISCUSSION THREAD

**Discussion by leimao** [ppoffice/hexo-theme-icarus#1044] [Q&A] [answered]
Large tables in the blog post will go outside screen on iPhone. Do we have a fix for that? Thank you.
Example on iPhone 13 Pro:
https://leimao.github.io/blog/Python-Concurrency-High-Level/

**Comment by leimao** [comment]
@ppoffice If you think this is a bug, I can file one formally on GitHub Issues.

**Comment by ppoffice** [comment, marked as answer]
```html
<div class="table-overflow">

// your table ...

</div>
```

**Reply by leimao** [reply]
This is perfect. Thank you very much.
